invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js

```diff
@@ -45,9 +45,11 @@ const queryBuilder = () => {
 
 const list = query => {
   return http.get(`${documentURL}?q=${query}`).then(response => {
+    let totalHits = response.data.hits.total ? response.data.hits.total : 0;
     response.data = response.data.hits.hits.map(hit =>
       serializer.fromJSON(hit)
     );
+    response.data.totalHits = totalHits;
```

Comment:
Even if I understand it is the easiest solution, I am personally not super fan of enhancing the Array and adding a custom property, it can lead to nasty issues (like [iteration issues](https://stackoverflow.com/questions/9952126/add-property-to-javascript-array)).

I would rather have:
```
response: {
    data: {
        hits: [],
        total: 245
    }
}
```
but then you will have to change more things, because the object structure is changing.

Let's see what the other reviewers think about it.

Reply:
I thought about it as the easiest solution, in the other case we would have to change structure passed in all the reducers.
There shouldn't be iteration issues if we won't use the `for ... in`.